### PR TITLE
docs: grammar fix pages 100 - 120

### PR DESCRIPTION
**Pull request by sh-rp**
### Description
docs: grammar fix pages 100 - 120

### CODE REVIEW

File: docs/website/docs/dlt-ecosystem/verified-sources/sql_database/advanced.md [hunk 1]
```diff
@@ -136,18 +134,19 @@ source = sql_database(
...
+This is particularly useful with `sql_table()` because you can maintain a separate configuration for each table (below we show **secrets.toml** and **config.toml**; you are free to combine them into one):
 
 The examples below show how you can set arguments in any of the `.toml` files (`secrets.toml` or `config.toml`):
 1. Specifying connection string:
     ```toml
     [sources.sql_database]
     credentials="mssql+pyodbc://loader.database.windows.net/dlt_data?trusted_connection=yes&driver=ODBC+Driver+17+for+SQL+Server"
     ```
-2. Setting parameters like backend, chunk_size, and incremental column for the table `chat_message`:
+2. Setting parameters like backend, chunk size, and incremental column for the table `chat_message`:
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/dlt-ecosystem/verified-sources/sql_database/advanced.md:149]:
```suggestion
2. Setting parameters like backend, `chunk_size`, and incremental column for the table `chat_message`:
```

File: docs/website/docs/dlt-ecosystem/verified-sources/rest_api/basic.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ After you get the token, add it to the `secrets.toml` file:
 github_token = "your_github_token"
 ```
 
-## Run the pipeline
+### Run the pipeline
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/dlt-ecosystem/verified-sources/rest_api/basic.md:113]:
```suggestion
## Run the pipeline
```

File: docs/website/docs/dlt-ecosystem/verified-sources/rest_api/basic.md [hunk 3]
```diff
@@ -130,11 +130,13 @@ github_token = "your_github_token"
    dlt pipeline rest_api show
    ```
 
-## Source configuration
+### Source configuration
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/dlt-ecosystem/verified-sources/rest_api/basic.md:133]:
```suggestion
## Source configuration
```

File: docs/website/docs/dlt-ecosystem/verified-sources/rest_api/basic.md [hunk 4]
```diff
@@ -206,14 +208,14 @@ def load_github() -> None:
...
 The declarative resource configuration is defined in the `config` dictionary. It contains the following key components:
 
-1. `client`: Defines the base URL and authentication method for the API. In this case it uses token-based authentication. The token is stored in the `secrets.toml` file.
+1. `client`: Defines the base URL and authentication method for the API. In this case, it uses token-based authentication. The token is stored in the `secrets.toml` file.
 
 2. `resource_defaults`: Contains default settings for all [resources](#resource-configuration). In this example, we define that all resources:
     - Have `id` as the [primary key](../../../general-usage/resource#define-schema)
     - Use the `merge` [write disposition](../../../general-usage/incremental-loading#choosing-a-write-disposition) to merge the data with the existing data in the destination.
-    - Send a `per_page` query parameter with each request to 100 to get more results per page.
+    - Send a `per_page` query parameter with each request to get more results per page.
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/dlt-ecosystem/verified-sources/rest_api/basic.md:216]:
```suggestion
    - Send a `per_page` query parameter with each request to 100 to get more results per page.
```

File: docs/website/docs/dlt-ecosystem/verified-sources/sql_database/advanced.md [hunk 5]
```diff
@@ -171,13 +170,13 @@ The examples below show how you can set arguments in any of the `.toml` files (`
...
 
-    The resulting source created below will extract data using **pandas** backend with **chunk_size** 1000. The table **chat_message** will load data incrementally using **updated_at** column. All the other tables will not use incremental loading, and will instead load the full data.
+    The resulting source created below will extract data using the **pandas** backend with **chunk_size** 1000. The table **chat_message** will load data incrementally using the **updated_at** column. All the other tables will not use incremental loading and will instead load the full data.
 
     ```py
     database = sql_database()
     ```
 
-You'll be able to configure all the arguments this way (except adapter callback function). [Standard dlt rules apply]((/general-usage/credentials/setup).
+You'll be able to configure all the arguments this way (except the adapter callback function). [Standard dlt rules apply](/general-usage/credentials/setup).
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/dlt-ecosystem/verified-sources/sql_database/advanced.md:179]:
```suggestion
You'll be able to configure all the arguments this way (except the adapter callback function). [Standard dlt rules apply](../../../general-usage/credentials/setup).
```

File: docs/website/docs/reference/command-line-interface.md [hunk 6]
```diff
@@ -9,37 +9,37 @@ keywords: [command line interface, cli, dlt init]
...
 ```sh
 pip install "dlt[cli]"
 ```
-that will add additional packages to current environment.
+that will add additional packages to the current environment.
 
 > 💡 We ask you to install those dependencies separately to keep our core library small and make it work everywhere.
 
-### github-action
+### GitHub action
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/reference/command-line-interface.md:42]:
```suggestion
### `github-action`
```

File: docs/website/docs/reference/command-line-interface.md [hunk 7]
```diff
@@ -56,9 +56,9 @@ schedule into quotation marks as in the example above.
 For the chess.com API example above, you could deploy it with
 `dlt deploy chess.py github-action --schedule "*/30 * * * *"`.
 
-Follow the guide on [how to deploy a pipeline with Github Actions](../walkthroughs/deploy-a-pipeline/deploy-with-github-actions) to learn more.
+Follow the guide on [how to deploy a pipeline with GitHub Actions](../walkthroughs/deploy-a-pipeline/deploy-with-github-actions) to learn more.
 
-### airflow-composer
+### Airflow composer
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/reference/command-line-interface.md:61]:
```suggestion
### `airflow-composer`
```

File: docs/website/docs/reference/telemetry.md [hunk 8]
```diff
@@ -120,25 +112,21 @@ Example for `load` pipeline run step:
 }
 ```
 
-## The message `context`
+## The message context
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/reference/telemetry.md:115]:
```suggestion
## The message `context`
```

File: docs/website/docs/reference/telemetry.md [hunk 9]
```diff
@@ -120,25 +112,21 @@ Example for `load` pipeline run step:
...
 
 The message context contains the following information:
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/reference/telemetry.md:117]:
```suggestion
The message `context` contains the following information:
```

File: docs/website/docs/running-in-production/alerting.md [hunk 10]
```diff
@@ -54,9 +54,9 @@ for package in info.load_packages:
     # Iterate over each table in the schema_update of the current package
     for table_name, table in package.schema_update.items():
         # Iterate over each column in the current table
-        for column_name, column in table["columns"].items():
+        for column_name, column in table["columns"]. items():
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/running-in-production/alerting.md:57]:
```suggestion
        for column_name, column in table["columns"].items():
```

**Review by AstrakhantsevaAA — COMMENTED** [review]
Amazing checker! Few comments:
1. One broken link (that's why deploy preview failed)
2. He tried to change the structure of the document several times (replaced `##` to`###`)
3. Checker does not always change titles correctly (I changed it back)

File: docs/website/docs/reference/performance.md [hunk 11]
```diff
@@ -19,50 +19,50 @@ can be replaced with:
...
 processing.
 
-### Controlling intermediary files size and rotation
-`dlt` writes data to intermediary files. You can control the file size and the number of created files by setting the maximum number of data items stored in a single file or the maximum single file size. Keep in mind that the file size is computed after compression was performed.
-* `dlt` uses a custom version of [`jsonl` file format](../dlt-ecosystem/file-formats/jsonl.md) between the **extract** and **normalize** stages.
-* files created between the **normalize** and **load** stages are the same files that will be loaded to destination.
+### Controlling intermediary file size and rotation
+`dlt` writes data to intermediary files. You can control the file size and the number of created files by setting the maximum number of data items stored in a single file or the maximum single file size. Keep in mind that the file size is computed after compression has been performed.
+* `dlt` uses a custom version of the [`jsonl` file format](../dlt-ecosystem/file-formats/jsonl.md) between the **extract** and **normalize** stages.
```
Comment (burnash) [inline docs/website/docs/reference/performance.md:39]:
```suggestion
* `dlt` uses a custom version of the [JSONL file format](../dlt-ecosystem/file-formats/jsonl.md) between the **extract** and **normalize** stages.
```

**Pull request merged**